### PR TITLE
Refactor/UI

### LINKED ISSUE #30: style/change-font
We have a default font right now, however, we had couple more options to choose from to adjust our style and match the wireframe.

### LINKED ISSUE #53: mobile responsive
User should be able to comfortably view application on mobile screen size 500px

### PR DESCRIPTION
**Pull request by coopterrones**
#### What's this PR do?
-adds mobile responsive to all views in the app

#### What does it fix?
-test fixes

#### Where should the reviewer start?
-line 1 all active `.scss` files

#### How should this be manually tested?
-check using dev tools mobile resize 
-check for all passing tests

### CODE REVIEW

**Review by yiranli624 — APPROVED** [review]
Nice refactor! We will reconstruct the html structure in postingview component

File: src/Components/User/User.scss [hunk 1]
```diff
@@ -209,4 +238,31 @@
...
+    border: 1px solid $main-orange;
+    padding: 0.5em 1em;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 12px;
+    width: 50vw;
+    margin: 0.5em;
+    transition: all 0.3s ease-in-out;
+    &:hover {
```
Comment (okalimbet) [inline src/Components/User/User.scss:270]: nice change, it definitely is better with cursor

File: src/Components/Loading/Loading.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.dot {
+  height: 40px;
+  width: 40px;
+  margin: 1em;
+  border-radius: 50%;
+  animation: scaling 2.5s ease-in-out infinite;
+}
+
+@keyframes scaling {
```
Comment (okalimbet) [inline src/Components/Loading/Loading.scss:20]: great keyframes! Love it!

**Review by okalimbet — APPROVED** [review]
Great fix for the UI, definitely solved a lot of bugs, we need to avoid merging different branches to keep it clean

**Pull request merged**